purchase_partner_selectable_option/readme/USAGE.rst

```diff
@@ -2,5 +2,4 @@ To use this module, you need to:
 
 #. Go to **Contacts > Contacts**
 #. Create a record and uncheck the "Selectable in orders" box on the Purchase part of the "Sales & Purchases" tab.
-#. Go to **Purchase > Orders > Request for Quotations**
-#. Create a record and the previously created partner will not appear in the Vendor field.
+#. Create a record and uncheck the "Selectable in purchase orders" box on the Purchase part of the "Sales & Purchases" tab.
```

Comment:
This is not correct. You must do the change on the other sentence, not to remove these ones and repeat the previous sentence with this change. Even more, I would put the first 2 steps in CONFIGURE.rst